.env.sample

```diff
@@ -6,6 +6,7 @@ NODE_ENV=development
 TZ=Europe/London
 LOG_LEVEL=info
 REDACT_LOGS=false
+SINGLE_LINE_LOG_FORMAT=true
```

Comment:
We'll need to raise a ticket for @abhi-markan to make the environment variable changes :)

Reply:
Raised APIM-578